src/scss/components/_targid.scss

```
@@ -268,7 +268,7 @@ div.filmstrip {
       }
     }
 
-    > .view {
+    > .viewss {
```

```suggestion
    > .views {
```

Was it a typo or intentionally?